mission-control/docs/config-db/scrapers/kubernetes.md

```diff
@@ -111,3 +111,24 @@ There are 3 different ways to specify which value to use when finding related co
 | `expr`  | Use an expression to get the value | `string` |          |
 | `value` | Specify a static value             | `string` |          |
 | `label` | Get the value from a label         | `string` |          |
+
+
+## Special annotations
+
+Kubernetes resources can be annotated with some special annotations that can direct the scraper to certain behaviors.
+
+| Annotation                                                    | Description                                                                |
+| ------------------------------------------------------------- | -------------------------------------------------------------------------- |
+| `config-db.flanksource.com/tags: "key1:val1,key2:val2"`       | Attach custom tags to the object. A config can have as many as `5` tags, so keep the custom tags limited.                                          |
+| `config-db.flanksource.com/ignore: true`                      | Exclude the object from being scraped along with all of its changes.       |
+| `config-db.flanksource.com/ignore-changes: <pattern>`         | Exclude changes by type for the given object that matches the pattern.     |
+| `config-db.flanksource.com/ignore-change-severity: <pattern>` | Exclude changes by severity for the given object that matches the pattern. |
+
+### Pattern matching
```

Comment:
Can we move this to Pattern in reference/common and then add a schema column to the table above - to make it look like the others. 

Then add a couple examples - including excluding Argo events